src/popover.ts

```diff
@@ -35,6 +35,86 @@ function patchSelectorFn<K extends string>(
 
 const nonEscapedPopoverSelector = /(^|[^\\]):popover-open\b/g;
 
+const styles = `
+  [popover] {
+    position: fixed;
+    z-index: 2147483647;
+    inset: 0;
+    padding: 0.25em;
+    width: fit-content;
+    height: fit-content;
+    border-width: initial;
+    border-color: initial;
+    border-image: initial;
+    border-style: solid;
+    background-color: canvas;
+    color: canvastext;
+    overflow: auto;
+    margin: auto;
+  }
+
+  [popover]:is(dialog[open]) {
+    display: revert;
+  }
+
+  [anchor].\\:popover-open {
```

Comment:
Originally this pseudo class was `:open`, and so I opted for `.:open` to provide a classname that would be different enough that it wouldn't conflict with existing styles. The pseudo class has since shifted to `:popover-open` and so we followed up with a name change, but even so [`.popover-open` seems to be in common enough usage that it felt best to stick with the `:` prefix](https://github.com/search?q=.popover-open&type=code).